testsuite/smoke-test/src/network.rs

```diff
@@ -22,6 +22,85 @@ use std::{
     time::{Duration, Instant},
 };
 
+#[ignore]
+#[tokio::test]
+async fn test_netperf_client() {
```

Comment:
I guess this test isn't complete? e.g., it looks like a (partial?) clone of `test_connection_limiting`?